app/ui/browser/views/navbar/location.jsx

```diff
@@ -111,7 +110,10 @@ export class Location extends Component {
   }
 
   componentDidMount() {
-    this.props.ipcRenderer.on('focus-urlbar', () => this.refs.input.select());
+    this.props.ipcRenderer.on('focus-url-bar', () => {
```

Comment:
I feel this should go through the normal channels as the other ipc messages and actions and showURLBar should be on the redux state